src/TouristInfoCards.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+function TouristInfoCards() {
```

Comment:
I would personally stick to one way to declare react functional components, so this is more of a consistency/nitpick comment

https://github.com/CodeYourFuture/cyf-hotel-react/pull/573#discussion_r1215415298